source/opentelemetry-collector-contrib/examples/tracing/ot-ai.yaml

```diff
@@ -0,0 +1,119 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
```

Comment:
similarly here .. we should have one version of the yaml which covers the omsagent + ot collector.